tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/RedirectManagement.spec.ts

```diff
@@ -0,0 +1,118 @@
+﻿import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+// Remove smoke tag before merging
+test.describe('Redirect Management tests', {tag: '@smoke'}, () => {
+  const disableStatus = 'Disabled';
+  const enableStatus = 'Enabled';
+  let documentTypeId = '';
+  let contentId = '';
+  const contentName = 'TestContent';
+  const documentTypeName = 'TestDocumentType';
+  const updatedContentName = 'UpdatedContentName';
+
+  test.beforeEach(async ({umbracoApi, umbracoUi}) => {
+    await umbracoApi.redirectManagement.setStatus(enableStatus);
+    await umbracoUi.goToBackOffice();
+    // Create a content
+    await umbracoApi.documentType.ensureNameNotExists(documentTypeName); 
+    documentTypeId = await umbracoApi.documentType.createDefaultDocumentTypeWithAllowAsRoot(documentTypeName);
+    contentId = await umbracoApi.document.createDefaultDocument(contentName, documentTypeId);
+    // Publish the content
+    await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.content.clickActionsMenuForContent(contentName);
+    await umbracoUi.content.clickPublishButton();
+  });
+
+  test.afterEach(async ({umbracoApi}) => {
+    await umbracoApi.redirectManagement.setStatus(enableStatus);
+    await umbracoApi.document.ensureNameNotExists(contentName);
+    await umbracoApi.documentType.ensureNameNotExists(documentTypeName);
+  });
+
+  test('can disable URL tracker', async ({umbracoApi, umbracoUi}) => {
+    // Act
+    await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.redirectManagement.clickRedirectManagementTab();
+    await umbracoUi.redirectManagement.clickDisableURLTrackerButton();
+    await umbracoUi.redirectManagement.clickDisableButton();
+
+    // Assert
+    // Verfiy that if renaming a published page, there are no redirects have been made
+    // rename the published content 
+    await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.content.openContent(contentName);
+    await umbracoUi.content.enterContentName(updatedContentName);
+    await umbracoUi.content.clickSaveAndPublishButton();
```

Comment:
Maybe rename the content by using the API

Reply:
If I rename the content by using the API, there is no redirects have been made. The redirect URL will be made when I rename it in UI.